tests/test_core.py

```diff
@@ -8336,6 +8336,35 @@ def test_emscripten_stack(self):
     self.set_setting('TOTAL_STACK', 4 * 1024 * 1024)
     self.do_run_in_out_file_test('tests', 'core', 'test_stack_get_free.c')
 
+  # Tests Settings.ABORT_ON_WASM_EXCEPTIONS
+  def test_abort_on_exceptions(self):
+    self.set_setting('ABORT_ON_WASM_EXCEPTIONS', 1)
+    self.emcc_args += ['--bind', '--post-js', 'post.js']
+    create_test_file('post.js', '''
+      addOnPostRun(function() {
+        try {
+          // Crash the program
+          _crash();
+        }
+        catch(e) {
+          // Catch the abort
+          out(true);
+        }
+
+        out("again");
+
+        try {
+          // Try executing some function again
+          _crash();
+        }
+        catch(e) {
+          // Make sure it failed with the expected exception
+          out(e === "program has already aborted!" || e === "DEAD");
```

Comment:
```suggestion
          out(e === "program has already aborted!");
```
I think we don't need the DEAD version now?